src/hooks/navigator.d.ts

```diff
@@ -0,0 +1,3 @@
+interface Navigator {
```

Comment:
can we remove this ? if not move this to a more appropriate place

Reply:
Will move. This prevents a raised exception in IE.